src/main/java/com/hillel/zakushniak/service/QuestionService.java

```diff
@@ -0,0 +1,52 @@
+package com.hillel.zakushniak.service;
+
+import com.hillel.zakushniak.model.Question;
+import com.hillel.zakushniak.repository.dao.QuestionRepository;
+
+import java.util.Iterator;
+import java.util.List;
+import java.util.Random;
+
+
+public class QuestionService {
+    public final QuestionRepository questionRepository;
+
+    public static final Random random = new Random();
+    private List<Question> allByRandomOrder;
+    private Iterator<Question> iterator;
```

Comment:
better make it method var

Reply:
sorry, could you please explain or give an example?